src/renderer/components/map/MapComponent.tsx

```diff
@@ -1,4 +1,5 @@
 import { DeltaBackend, sendMessageParams } from '../../delta-remote'
+import { C } from 'deltachat-node'
```

Comment:
must be
```suggestion
import { C } from 'deltachat-node/dist/constants'
```
because otherwise frontend bundler thinks it needs to bundle nodejs stuff